### DISCUSSION TITLE
Change Notes - v4

### DISCUSSION THREAD

**Discussion by RoboZoom** [ngneat/transloco#561] [General]
Hi - are there release notes posted for v4? 

 I checked the `BREAKING_CHANGES.md` document and there is only information from v3.  Does that mean that there are no breaking changes for v4?  Thanks!

**Comment by shaharkazaz** [comment]
@RoboZoom The changelog in the project root is the old one, [here](https://github.com/ngneat/transloco/blob/master/libs/transloco/CHANGELOG.md) is the correct changelog, when I'll have the time I'll fix this.
TL;DR - v4 depends on Angular v13+

**Reply by RoboZoom** [reply]
Thanks!
